### DISCUSSION TITLE
Chainlink price data not fetching says this

### DISCUSSION THREAD

**Discussion by Asharp19** [smartcontractkit/full-blockchain-solidity-course-py#530] [Q&A]
[](url
<img width="632" alt="dsda" src="https://user-images.githubusercontent.com/95505745/145517477-c03e64f5-4170-4c69-a859-4ffcee2c2672.png">
)

I used the code on the Chainlink's site that redirects to remix IDE.
![dada](https://user-images.githubusercontent.com/95505745/145517624-3913fe8d-3671-40f6-856b-8da4dea8141e.JPG)

**Comment by cromewar** [comment]
Hello @Asharp19 this happens because you are trying the deploy a interface which is abstract, so on Remix please be sure to select the correct contract for deployment:

You should not deploy de AggregatorV3Interface
![image](https://user-images.githubusercontent.com/20199180/145590434-cb845c64-13d3-46a7-bc45-8361b0cf1d1b.png)
